router/build.gradle.kts

```diff
@@ -22,7 +22,7 @@ android {
 }
 
 dependencies {
-  implementation("androidx.appcompat:appcompat:${versions.supportLib}")
+  implementation("androidx.appcompat:appcompat:${versions.appcompat}")
```

Comment:
Why were we using supportLib before?

Reply:
No particular reason, it was there before and we must have forgot to switch to `appcompat` version